.github/workflows/scripts.yaml

```diff
@@ -44,12 +44,6 @@ jobs:
     steps:
       - uses: actions/checkout@master
 
-      - name: Reclaim free space!
```

Comment:
Why isn't it needed anymore? :?

Reply:
I'd guess the Shipyard jobs use less space than the jobs that build on Shipyard and do more stuff. :man_shrugging:

Reply:
I actually found that by default when running KIND, each node (which runs in a docker container) has a volume allocated with the same size as the image, so for our current image it's about 1.3GB per node, so for shipyard that only had 2 clusters with 2 nodes each it's roughly 1.3*2*2=5.2GB needed, which is much less than the 10GB that GH actions runners guarantee (but in reality have even more space since they leave headroom).